pom.xml

```diff
@@ -120,6 +120,8 @@
         <version.plugin.wiremock>2.14.0</version.plugin.wiremock>
         <version.plugin.echo>0.4.0</version.plugin.echo>
 
+        <version.helidon-cli>2.1.0</version.helidon-cli>
```

Comment:
Is the `cli` released separately? If not, this should be the same version as `build-tools`

Reply:
No change was made to the CLI plugin, so he left this version the same.

Reply:
The CLI wasn't released as part of 2.1.1 because no changes were made to the CLI ; thus the difference in versions.